android/app/src/main/java/app/organicmaps/MapFragment.java

```diff
@@ -43,7 +43,7 @@ public void updateMyPositionRoutingOffset(int offsetY)
 
   public void destroySurface()
   {
-    mMap.onSurfaceDestroyed(requireActivity().isChangingConfigurations(), isAdded());
+      mMap.onSurfaceDestroyed(true, isAdded());
```

Comment:
Please add `boolean activityIsChangingConfigurations` parameter to destroySurface() function and pass it from other calling functions.

Reply:
And there is no need to change the indentation ;-)